exporter/awsemfexporter/metric_translator.go

```diff
@@ -467,3 +499,15 @@ func translateGroupedMetricToEmf(groupedMetric *groupedMetric, config *Config, d
 
 	return event, nil
 }
+
+func filterAWSEMFAttributes(labels map[string]string) map[string]string {
+	// remove any labels that are attributes specific to AWS EMF Exporter
+	filteredLabels := make(map[string]string)
+	emfAttributeFilterRE := regexp.MustCompile(emfAttributeFilter)
+	for labelName := range labels {
+		if !emfAttributeFilterRE.MatchString(labelName) {
+			filteredLabels[labelName] = labels[labelName]
+		}
+	}
```

Comment:
I'm concerned about the performance impact of applying regex matching on every metric attribute for each grouped metric data point. 
Is there any requirement to add more `aws.emf.*` attribute patterns? Could you perform an exact match for `labelname == "aws.emf.storage_resolution"` and break the loop once it's found?